libs/ui/src/containers/InlineBanner/shared/utils.ts

```diff
@@ -0,0 +1,41 @@
+import React, { ReactNode } from 'react'
+
+import SVG from 'react-inlinesvg'
+
+import { Icon, IconType } from '../../../pure/Icon'
+import { StatusColorEnums, StatusColorVariant } from '../../../theme/statusColors'
+
+export function renderIcon(
```

Comment:
Render functions are not preferrable due to performance reasons, it's better to make it a component